game.py

```diff
@@ -0,0 +1,90 @@
+# Here is the description of the game simulation.
+
+# this is a two player card game
+# the game starts with a deck of cards
+# the cards are dealt out to both players
+# on each turn:
+# both players turn over their top-most card
+# the player with the higher valued card takes the cards and puts them in their scoring pile (scoring 1 point per card)
+# this continues until the players have no cards left
+# the player with the highest score wins
+# It’s considered a simulation because the players don’t have any choice, dont worry about input
+import random
+
+class Card:
+    def __init__(self):
+        self.types = ['spade', 'heart', 'diamond', 'club']
+        self.cards = []
+
+    def suffle_card(self):
+        indexs = [i for i in range(1,14)]
+        # shuffle the value first
+
+        # give the shuffle deck card
+        for i in indexs:
+            for type in self.types:
+                self.cards.append((type, i))
+        random.shuffle(self.cards)
+
+
+class Player:
+    def __init__(self):
+        self.cards = []
+        self.score = 0
+
+
+class Game:
+    def __init__(self):
+        self.player1 = Player()
+        self.player2 = Player()
+    
+    def get_cards(self):
+        cur_cards = Card()
```

Comment:
The concept of `Card` class is a bit confusing. I guess you're trying to define a `CardDeck` here. However, you still need another `Card` class. I would suggest the following changes:
1. `types` should be in `Card` not `CardDeck`. 
2.  All cards related logic should live in the `Card` class. For example, compare logic should be one of the functions in `Card`. You only need to write `card_one > card_two` in your `Game`.

Reply:
`Card` class should only represent a single card, e.g., Spade 9. If you wnt several cards, just create a list or even follow @1998Isabel 's suggestion to implement CardDeck.